src/io_uring/src/lib.rs

```diff
@@ -90,24 +98,48 @@ impl IoUring {
             len if fd < 0 || (len as i32 - 1) < fd => {
                 Err((Error::InvalidFixedFd(fd), op.user_data()))
             }
-            _ => self.squeue.push(unsafe { op.into_sqe() }).map_err(
-                |err_tuple: (SQueueError, T)| -> (Error, T) {
+            _ => self
+                .squeue
+                .push(unsafe { op.into_sqe() })
+                .map(|_| {
+                    self.to_submit = self.to_submit.saturating_add(1);
```

Comment:
How likely is is that we will get to saturating this value? Should we treat this in a separate error case or is it actually impossible to get to `u32::MAX` here?

Reply:
it's impossible to get u32::MAX here. I added comments.